install/kubernetes/helm/istio/values-istio-all.yaml

```diff
@@ -22,3 +22,6 @@ servicegraph:
 
 prometheus:
   enabled: true
+
+zipkin-to-stackdriver:
+  enabled: true
```

Comment:
do we want this enabled by default? (I don't know enough about the addons to intelligently answer the question myself)

Reply:
by default it is false, it is set in https://github.com/SpringUpenn/istio/commit/b4b25f92f9a3140d60b1ce65f4a63e4491e4c820#diff-7dad29cba9d2ca3e8570c8f65f4b7e86, this is for values-istio-all.yaml, which wants to enable all addons(based on it sets true for zipkin, grafana etc.), correct me if I understand wrong.